Makefile

```diff
@@ -52,6 +52,10 @@ ginkgo:
 deps-update:
 	go mod tidy
 
+e2e-test:
+	go build -o apply_rules test/firewall_run.go 
+	./apply_rules
```

Comment:
If we add more tests to the e2e suite and would like to focus on one the rules will always be applied and might break it